maturity-level-one/README.md

```diff
@@ -150,20 +150,28 @@ Every API should have documentation in the OpenAPI format. If you want to genera
 
 Here is an example on how to generate them with Swashbuckle
 ```csharp
-/// <summary>
-///     Get Health
-/// </summary>
-/// <remarks>Gets the current health status of the API</remarks>
-[HttpGet]
-[Route("health")]
-[SwaggerOperation("Health_Get")]
-[SwaggerResponse(HttpStatusCode.OK, "API is up & running")]
-[SwaggerResponse(HttpStatusCode.InternalServerError, "API is not available")]
-public IHttpActionResult Get()
-{
-    return Ok();
-}
+        /// <summary>
+        /// Get the profiles of the players
```

Comment:
Pretty long Operation name, no?

```suggestion
        /// Get Player Profiles
```

or

```suggestion
        /// Get Players
```

Reply:
This is not the operation name, it's the description

Reply:
It's actually not, summary is operation name (not to confuse with operation id).
Think the sample is running on an old version of Swashbuckle then.

See https://blog.tomkerkhove.be/2018/04/13/changes-to-azure-api-management-openapi/